clang/test/dpct/cmake_migation/case_013/expected.txt

```diff
@@ -0,0 +1,12 @@
+cuda_compile_cubin(cubinfile_1 ${cufile})
```

Comment:
should line 1, 2 7, 8 be migrated to dpct_compile_sycl_code ?

Reply:
As per the discussion [here](https://github.com/oneapi-src/SYCLomatic/pull/1476#discussion_r1420119550), we wanted to migrate **cuda_compile_cubin** macro to **dpct_compile_sycl_code** for creating a **_.o_** file in place of a **_.cubin_** file